docs/discover/context.asciidoc

```diff
@@ -15,13 +15,26 @@ image::images/Expanded-Document.png[Expanded Document]
 The context view displays a number of documents before and after the anchor 
 document. The anchor document itself is highlighted in blue. The view is sorted 
 by the time field specified in the index pattern configuration and uses the 
-same set of columns as the Discover view the context was opened from.
+same set of columns as the Discover view the context was opened from. If there
+are multiple documents with the same time field value, the internal document
+order is used as a secondary sorting criterion by default.
+
+NOTE: The field used for tiebreaking in case of equal time field values can be
+configured using the advanced setting `context:tieBreakerFields` in
+<<advanced-options, *Management > Advanced Settings*>>, which defaults to the
+`_doc` field. The value of this setting can be a comma-separated list of field
+names, which will be checked in sequence for suitability when a context is
+about to be displayed. The first suitable field is then used as the tiebreaking
+field. A field is suitable if the field exists and is sortable in the index
+pattern the context is based on. Examples for common suitable fields include
+log line numbers, monotonically increasing counters and high-precision
+timestamps.
```

Comment:
@weltenwort I understand that all fields that support doc_values have them enabled by default but is it worth pointing out that we do not recommend using a field as a tiebreaker that does not support doc_values? We could link to the documentation for doc_values as well.